config/locales/en/brexit_landing_page.yml

```diff
@@ -15,15 +15,15 @@ en:
         list_block: |
           Check what you need to do if you:
           <ul>
-            <li><a href="/prepare-import-to-uk-after-brexit" data-ecommerce-row="true" data-ecommerce-path="/prepare-import-to-uk-after-brexit">import goods from the EU</a></li>
-            <li><a href="/prepare-export-from-uk-after-brexit" data-ecommerce-row="true" data-ecommerce-path="/prepare-export-from-uk-after-brexit">export goods to the EU</a></li>
-            <li><a href="/transport-goods-from-uk-by-road" data-ecommerce-row="true" data-ecommerce-path="/prepare-export-from-uk-after-brexit">transport goods out of the UK by road</a></li>
+            <li><strong><a href="/prepare-import-to-uk-after-brexit" data-ecommerce-row="true" data-ecommerce-path="/prepare-import-to-uk-after-brexit">import goods from the EU</a></strong></li>
```

Comment:
Using `<strong>` here doesn't feel right to me. Could we use a class instead, e.g: `govuk-!-font-weight-bold`

Reply:
done, amended